src/sentry_plugins/github/plugin.py

```diff
@@ -336,8 +336,17 @@ def compare_commits(self, repo, start_sha, end_sha, actor=None):
         client = self.get_client(actor)
         # use config name because that is kept in sync via webhooks
         name = repo.config['name']
-        try:
-            res = client.compare_commits(name, start_sha, end_sha)
-        except Exception as e:
-            self.raise_error(e)
-        return [{'id': c['sha'], 'repository': repo.name} for c in res['commits']]
+        if start_sha is None:
+            try:
+                res = client.get_last_commits(name, end_sha)
+            except Exception as e:
+                self.raise_error(e)
+            else:
+                return [{'id': c['sha'], 'repository': repo.name} for c in res]
```

Comment:
let's maybe limit this to 10 commits for now? kind of arbitrary, but i'm thinking of this mostly as a way for people to make sure their setup is correct